### DISCUSSION TITLE
clickable stat table item

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1299] [Q&A]
is there a particular reason why a `stat_table_item()` is not clickable / submittable like a `stat_list_item()`?

Also, the documentation for a `stat_table_card()` and  `stat_list_card()` has optional links but I couldn't find how to use this.

Essentially: what I'm trying to implement is - clickable stat items / dataset names, to view more info about them

 
<img width="1785" alt="image" src="https://user-images.githubusercontent.com/64787868/158689384-80f54694-98b2-4507-8bd7-54300acfbe32.png">

**Comment by azim-b** [comment]
https://wave.h2o.ai/blog/release-0.11.0/#stat-lists

**Comment by mtanco** [comment]
Items are clickable by using the `name` property. Any `ui.stat_table_item()` with a name will be clickable, any without a name will not be clickable. If the `ui.stat_table_card()` has a name then your q.args will look like: `q.args.name_of_card:name_of_clicked_item` if you do not include the `name` option on the card it will be `q.args.name_of_clicked_item:True`.

Here's an example:
```python
from h2o_wave import main, app, Q, ui


@app('/')
async def serve(q: Q):
    print(q.args)

    if not q.client.initialized:
        q.page["meta"] = ui.meta_card("")
        q.page["form"] = ui.stat_table_card(
            box='4 4 4 4',
            title='My Stat Table Card',
            name='my_stat_table_card',
            subtitle='This is a test.',
            columns=["Name", "Value", ""],
            items=[
                ui.stat_table_item(
                    label=f'Row {i}',
                    name=f'my_row_{i}' if i < 2 else "",
                    caption='This is a row',
                    icon='Play',
                    icon_color='Blue',
                    values=[str(i), ""])
                for i in range(3)],
            commands=[]
        )

        q.client.initialized = True

    if q.args.my_stat_table_card is not None:
        q.page["meta"].dialog = ui.dialog(
            title="You Clicked a Row",
            items=[
                ui.text(f"You clicked on the row named: {q.args.my_stat_table_card}")
            ],
            closable=True
        )



    await q.page.save()

```
